app/controllers/v0/form1010_ezr_attachments_controller.rb

```diff
@@ -1,16 +1,61 @@
 # frozen_string_literal: true
 
+require 'form1010_ezr/service'
+
 module V0
   class Form1010EzrAttachmentsController < ApplicationController
     include FormAttachmentCreate
     service_tag 'health-information-update'
 
     FORM_ATTACHMENT_MODEL = Form1010EzrAttachment
 
+    def create
+      validate_file_upload_class!
+      validate_file_type
+      save_attachment_to_cloud!
+      save_attachment_to_db!
+
+      render json: serializer_klass.new(form_attachment)
+    end
+
     private
 
     def serializer_klass
       Form1010EzrAttachmentSerializer
     end
+
+    # These MIME types correspond to the extensions accepted by enrollment system: PDF,WORD,JPG,RTF
+    def mime_subtype_allow_list
```

Comment:
Would this validation logic (`mime_subtype_allow_list` and `validate_file_type` make more sense to live in the ezr_attachment model, or some type of validator/helper service? I tend to like skinny controllers and this feels like it shouldn't be in here.

Reply:
Along these lines, making this more reusable could be beneficial, since it feels like it is trying to handle the situation when a user has manually updated a file to be named something that is not the actual file type. So I would expect we could have that issue elsewhere? Moving that out of this controller would help with that.

Reply:
Have we looked at performance for this? It feels like this could be not performant if we are validating this each time an attachment is added.